README.md

```diff
@@ -88,6 +88,18 @@ will be possible for all language constructs. There are various
 instructions that are not supported during verification. However,
 any failure during `llvm_load_module` should be considered a bug.
 
+## Notes on Rust
+
+SAW has experimental support for analyzing Rust programs. To do so, one must
+compile Rust code using [`mir-json`](https://github.com/GaloisInc/mir-json), a
+tool which compiles Rust code to a machine-readable, JSON-based format. Each
+version of SAW understands the JSON output of a particular version of
+`mir-json`, so make sure that you build the version `mir-json` that is included
+in the `mir-json` submodule (located in `deps/mir-json`).
+
+For more information on how to install `mir-json`, follow the instructions
+[here](https://github.com/GaloisInc/mir-json#installation-instructions).
+
```

Comment:
I feel like this section ought to say something about translate-libs and $SAW_RUST_LIBRARY_PATH, but maybe that belongs in a separate branch.

Otherwise, LGTM.

Reply:
I guess there's two aspects to this: (1) the translate-libs process should be documented somewhere, but also (2) all the critical pieces of the complete setup process for doing MIR verification should be mentioned in the same places, especially as long as things fail in a non-obvious way if you forget parts of it. So I think if we're going to mention setting up mir-json in the README, it should also mention the translate-libs process, even if just as a reference into the manual.

(I actually think it's probably good to have the complete setup process in the README, but I'm not totally committed to this given that the whole thing is experimental and hopefully by the time it isn't it'll all be less complicated and less fragile)

So... carry on I guess :-)

Reply:
I've pushed a new version of the PR that expands the discussion in the `README` somewhat, mentioning both the need for a specific `mir-json` version and the need for custom Rust standard libraries. I've also fleshed out the relevant section in the SAW manual to mention the `mir-json` submodule, `translate_libs.sh`, and the `SAW_RUST_LIBRARY_PATH` environment variable. I think this should be much better than it was before—PTAL.

The manual currently assumes that you are starting from a `saw-script` repo checkout. This is not ideal, since many SAW users download a binary distribution instead. In order to do better here, however, I think we need to first explicitly version `mir-json`'s schema. I've started work on that in https://github.com/GaloisInc/mir-json/pull/75.